oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/JdbcOAuth2AuthorizedClientService.java

```diff
@@ -181,8 +215,10 @@ public void removeAuthorizedClient(String clientRegistrationId, String principal
 		SqlParameterValue[] parameters = new SqlParameterValue[] {
 				new SqlParameterValue(Types.VARCHAR, clientRegistrationId),
 				new SqlParameterValue(Types.VARCHAR, principalName) };
-		PreparedStatementSetter pss = new ArgumentPreparedStatementSetter(parameters);
-		this.jdbcOperations.update(REMOVE_AUTHORIZED_CLIENT_SQL, pss);
+		try (LobCreator lobCreator = this.lobHandler.getLobCreator()) {
```

Comment:
Please revert back to `ArgumentPreparedStatementSetter` as reading/writing `BLOB` is not required on the remove.

Reply:
:+1: